wikimedia/wikipedia-ios
# Create account username error design review tweaks

**Phabricator:** https://phabricator.wikimedia.org/T347999

### Notes
Minor tweaks to prevent a jump found in [design review](https://phabricator.wikimedia.org/T347999#10354798).

### Test Steps
1. Log out, go to Create Account view.
2. Type "1.2.3.4" in username field
3. Confirm when error shows, the content below doesn't jump.
